inc/datahandlers/post.php

```diff
@@ -1879,6 +1879,20 @@ function update_post()
 
 				$db->update_query("threads", $this->thread_update_data, "tid='".(int)$post['tid']."'");
 			}
+
+			// Update any moved thread links to have corresponding new subject.
+			if(isset($post['subject']))
+			{
+				$query = $db->simple_select("threads", "tid, closed", "closed='moved|".$this->tid."'");
+				if($db->num_rows($query) > 0)
+				{
+					while($result = $db->fetch_array($query))
+					{
+						$update_data['subject'] = $db->escape_string($post['subject']);
```

Comment:
This definition can be moved outside the `while` loop, since values will be the same

Reply:
Changed that, thanks for your suggestion.